src/backend/winit.rs

```diff
@@ -52,6 +52,8 @@ pub struct WinitState {
 
 impl WinitState {
     pub fn render_output(&mut self, state: &mut Common) -> Result<()> {
+        println!("Render output");
```

Comment:
Isn't this super spammy? :sweat_smile:

Reply:
Lol. Naturally I added that when I was trying to figure out why nothing was showing up, and I guess I didn't remove it.